app/components/Dashboard/AssetBalancesPanel/AssetBalancesPanel.jsx

```diff
@@ -99,11 +107,35 @@ export default class AssetBalancesPanel extends React.Component<Props> {
     return neoPrice && NEO !== '0' ? neoPrice * toNumber(NEO) : 0
   }
 
+  getNEOFormattedPriceChange = (): string => {
+    return (this.props.neoPriceChange >= 0 ? '+' : '') + (this.props.neoPriceChange * 100).toFixed(2) + '%'
+  }
+
+  getNEOPriceChangeDirection = (): string => {
+    if (this.props.neoPriceChange < 0) {
+      return 'decrease'
+    } else {
+      return 'increase'
+    }
+  }
+
   getGASValue = (): number => {
     const { GAS, gasPrice } = this.props
     return gasPrice && GAS !== '0' ? gasPrice * toNumber(GAS) : 0
   }
 
+  getGASFormattedPriceChange = (): string => {
+    return (this.props.gasPriceChange >= 0 ? '+' : '') + (this.props.gasPriceChange * 100).toFixed(2) + '%'
```

Comment:
Better extract that logic to a function and use it in both `getGASFormattedPriceChange` and `getNEOFormattedPriceChange`.